partials/footer.hbs

```diff
@@ -1,3 +1,3 @@
-<footer>
+<footer class="container footer  col-xs-10 col-xs-offset-1">
```

Comment:
What does happen if a page has infinite scrolling for instance? 
Of course it is possible for different pages have different footer, if they are different section of a site the header and the footer, for instance, can render differently: different color, different size, etc etc